docs/ops/signals/IRDFT_9.md

```diff
@@ -0,0 +1,210 @@
+# Inverse Discrete complex-to-real Fourier Transformation (IRDFT) {#openvino_docs_ops_signals_IRDFT_9}
+
+**Versioned name**: *IRDFT-9*
+
+**Category**: *Signal processing*
+
+**Short description**: *IRDFT* operation performs the inverse complex-to-real discrete Fourier transformation of input tensor by specified dimensions.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+*   **1**: `data` - Input tensor of type *T* with data for the IRDFT transformation. Type of elements is any supported floating-point type. The last dimension of the input tensor must be equal to 2, that is the input tensor shape must have the form `[D_0, D_1, ..., D_{N-1}, 2]`, representing the real and imaginary components of complex numbers in `[:, ..., :, 0]` and in `[:, ..., :, 1]` correspondingly. **Required.**
+*   **2**: **2**: `axes` - 1D tensor of type *T_IND* specifying dimension indices where IRDFT is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, for example, `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`, `[-3, 0, -2]`. These indices should be integers from `-(r - 1)` to `(r - 2)` inclusively, where `r = rank(data)`. A negative axis `a` is interpreted as an axis `r - 1 + a`. Other dimensions do not change. The order of elements in `axes` attribute matters, and is mapped directly to elements in the third input `signal_size`. **Required.**
+*   **NOTE**: The following constraint must be satisfied: `rank(data) >= len(axes) + 1 and input_shape[-1] == 2 and (rank(data) - 1) not in axes and (-1) not in axes`.
```

Comment:
Unclear which axes should be passed based on -1 was original axes in the framework. How FrontEnd should act?

Reply:
As for TF, TF `IRFFTxD` operations have not any inputs specify axes to calculate. But axes are hardcoded in each such operation: `IRFFT1D` uses axis `-1` of input complex tensor, `IRFFT2D` uses axes `[-2, -1]` of the input complex tensor, and `IRFFT3D` uses axes `[-3, -2, -1]`. And TF has direct support of complex numbers, but we don't have such support, and, as a result, we need to emulate complex tensors of the shape `[D_0, ..., D_{N-1}]` using real tensors of the shape `[D_0, ..., D_{N-1}, 2]`: we consider any complex number `z = x + iy` as a tuple `(x, y)`. 

Consequently, we need to consider `-1` axis of our real tensor of the shape `[D_0, ..., D_{N-1}, 2]` as an axis corresponding to the dimension `D_{N-1}`, `-2` axis as an axis corresponding to the dimension `D_{N-2}`, and so on.

Hence, in MO we should use a transformation to set up the correct axes.

In TF FrontEnd, we should set up correct axes, when we create our `IRDFT` instead of TF operation `IRFFTxD`.

As for PyTorch, this framework uses real input tensors for `IRFFT`, but calculations interpret such tensors as complex. That is, PyTorch uses the same mean of negative axes, as we use.